pkg/util/throttle.go

```diff
@@ -26,9 +26,12 @@ type RateLimiter interface {
 	Accept()
 	// Stop stops the rate limiter, subsequent calls to CanAccept will return false
 	Stop()
+	// Saturation returns a percentage number which describes how saturated
+	// this rate limiter is.
```

Comment:
I probably wrote something too complex. Rate limiter has to 'limits' one that is sustainable and 'burst' limit for short bursts. I think that exposing `number of tokens / max number of tokens` (something like 'burst saturation') is not enough. We want to know when RL allowed more QPS than it's sustainable level. There's no difference between tokens themselves, but there's a difference between having 1 QPS and 10 QPS with RL producing 2tok/sec and having burst value of 20. Does it make more sense now?